src/tutorials/emx/index.js

```diff
@@ -0,0 +1,7 @@
+const files = require.context('.', false, /\.vue$/)
+const modules = {}
+files.keys().forEach((key) => {
+  if (key === './tutorials.ts') return
+  modules[key.replace(/(\.\/|\.vue)/g, '')] = files(key).default
```

Comment:
Cool, did not know you could do something like this. :)

Reply:
Me neither, I asked google :) I wanted to make everything as automagic as possible.